utils/ioprovenance.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+Utility funcitons for provenance of feature files.
+
+Authors: Phillip J. Wolfram
+Last Modified: 09/29/2016
+"""
+
+import os
+import sys
+import socket
+import subprocess
+import numpy as np
+import datetime
+
+def provenance_command(): #{{{
+    cwd = os.getcwd()
```

Comment:
I have minor privacy concerns about `cwd` being included in the history.  Do you feel that the usefulness of this info (which I perceive as being rather short lived) outweighs the fact that my directory structure becomes publicly available when I create geometric features?